distributions/standalone/assembly-whitelist.txt

```diff
@@ -348,7 +349,7 @@
 /mule-standalone-${productVersion}/lib/opt/wss4j-1.6.18.jar
 /mule-standalone-${productVersion}/lib/opt/xml-resolver-1.2.jar
 /mule-standalone-${productVersion}/lib/opt/xmlpull-1.1.3.1.jar
-/mule-standalone-${productVersion}/lib/opt/xmlschema-core-2.2.1.jar
+/mule-standalone-${productVersion}/lib/opt/xmlschema-core-2.1.0.jar
```

Comment:
the newer version was only introduced becayse the spring-config module was generating schemas through the xmlschema-core library 2.2.1 for XML extensions plugins.
This PR introduces the ExtensionModel to describe XML extensions, and the whole generation is no longer needed, thus, I fallback to the library mule was using before my changes.